qutebrowser/browser/commands.py

```diff
@@ -1515,6 +1515,7 @@ def show_source_cb(source):
             new_tab.data.viewing_source = True
 
         tab.dump_async(show_source_cb)
+        # pylint: enable=no-member
```

Comment:
You should be able to move the enable/disable here so it's only around the `pygments.lexers` / `pygments.formatters` calls. Also, while you're at it, please change the URL in the comment to `https://github.com/PyCQA/pylint/issues/491`